python/Ganga/GPIDev/Base/Proxy.py

```diff
@@ -924,6 +936,15 @@ def _getattribute(self, name):
 
     return type(name, (GPIProxyObject,), d)
 
+_metaClassHolder = None
+def _getMetaClass():
+    global _metaClassHolder
+    if _metaClassHolder is None:
+        from Ganga.GPIDev.Base.Objects import ObjectMetaclass
+        _metaClassHolder = ObjectMetaclass
+    return _metaClassHolder
```

Comment:
The problem here is perhaps coming from the import hierarchy of Proxy.py and Objects.py being backwards. Really, Objects.py should have no knowledge of proxies or anything to do with it, at all! `GangaObject`s should be able to operate in complete isolation of the proxy system. Proxy.py, however should absolutely know about `GangaObject`s since that is what they are wrapping and so Proxy.py should import Objects and not vice versa.